src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -137,6 +161,15 @@ public static boolean deleteFolder(final File file)
         return false;
     }
 
+    public static boolean deleteFile(final File file)
```

Comment:
it will be unused and it can't do damage. in the devel branch there is coreprotect handler.